backend/src/server.py

```diff
@@ -432,7 +432,17 @@ async def get_dependencies(_request: Request):
     await nodes_available()
     all_dependencies = []
     for package in api.registry.packages.values():
-        pkg_dependencies = [x.toDict() for x in package.dependencies]
+        pkg_dependencies = []
+        for pkg_dep in package.dependencies:
+            installed_version = installed_packages.get(pkg_dep.pypi_name, None)
+            pkg_dep_item = {
+                **pkg_dep.toDict(),
+            }
+            if installed_version is None:
+                pkg_dep_item["installed"] = False
```

Comment:
Why False?

Reply:
Because it's not installed. Idk, I guess it could be None instead